network/topics/scoring_test.go

```diff
@@ -0,0 +1,34 @@
+package topics
+
+import (
+	"testing"
+
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTruncateStats(t *testing.T) {
+	filtered := map[string]*pubsub.TopicScoreSnapshot{
+		"ssv.v2.103": {
+			TimeInMesh:               0,
```

Comment:
Can we test both zero and non-zero values for all the fields to test all of them?